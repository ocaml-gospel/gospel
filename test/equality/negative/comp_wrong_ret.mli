val f : int -> int -> string
(*@ comparison *)
