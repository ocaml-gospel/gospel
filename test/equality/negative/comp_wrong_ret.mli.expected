
*******************************
********** Parsed file ********
*******************************
val f : int -> int -> string[@@gospel {| comparison |}]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

val f : int -> int -> string
(*@ comparison
     *)
File "comp_wrong_ret.mli", lines 1-2, characters 0-27:
Error: Type checking error: type is incompatible with equality clause
